neuralmonkey/learning_utils.py

```diff
@@ -88,7 +88,8 @@ def training_loop(tf_manager: TensorFlowManager,
         runners_batch_size: batch size of runners. It is the same as batch_size
             if not specified
         initial_variables: variables used for initialization, for example for
-            continuation of training
+            continuation of training. Provide it with a path to your model and 
+            its variables.data without any suffix.
```

Comment:
Aby "without any suffix" nebylo spíš matoucí: když tam je třeba `variables.data.9` (když ukládáš víc checkpointů) nebo `variables.data.cont-3` (když jedeš do stejnýho output adresáře) - to jsou taky validní názvy.

Napsal bych třeba `checkpoint file group common prefix, e.g. "variables.data", or "variables.data.3" in case of multiple checkpoints per training run`

Reply:
trochu jsem to upravil, co ty na to?